lib/usbd/usbd_private.h

```diff
@@ -174,6 +179,11 @@ struct usbd_device {
 #endif
 };
 
+typedef enum {
+	  usbd_paShutdown
```

Comment:
- Enum labels are uppercase. (see coding style for full information)
- Without compelling reason, please dont use typedef. (see coding style for full information)
If you really want, you can later do `typedef enum usbd_power_action usbd_power_action`.